src/ToolBox/superpmi/superpmi/neardiffer.cpp

```diff
@@ -100,7 +100,19 @@ bool NearDiffer::InitAsmDiff()
             return false;
         }
 
-        corAsmDiff = (*g_PtrNewDiffer)(Target_Host, &CorPrinter, NearDiffer::CoreDisCompareOffsetsCallback);
+        TargetArch coreDisTargetArchitecture = Target_Host;
+#ifdef _TARGET_AMD64_
```

Comment:
Seems like we're getting a bunch of this type of `#ifdef`. Maybe we need to compute and capture all this information in one place, to avoid spreading the `#ifdefs` around.

Reply:
Yes, it is a good idea! I can clean this up next time when touching the code